pcdsdevices/lakeshore.py

```diff
@@ -0,0 +1,87 @@
+"""
+Classes for lakeshore temperature controller
+"""
+import logging
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+from ophyd import FormattedComponent as FCpt
+
+from .interface import BaseInterface
+
+logger = logging.getLogger(__name__)
+
+
+class Heater(BaseInterface, Device):
+    """
+    HeaterState Channel Object.
+
+    Parameters
+    ----------
+    channel_prefix : str
+        The EPICS base of the HeaterState Channel. E.g.: ``
+    """
```

Comment:
This docstring bit seems inaccurate, you have two init parameters which are `prefix` and `channel`, not one parameter named `channel_prefix`. The example seems incomplete too.

I think it would have been OK to omit this part or the whole docstring rather than have it be inaccurate, or you could correct it.